OG_Format.adoc

```diff
@@ -662,7 +676,8 @@ For each sensor it is created a group with the following group attributes:
 
 A parameter is a measurement of a physical phenomenon; it can be provided by a sensor (in sensor counts or in physical units) or computed (derived) from other parameters. A sensor can measure 1 to N parameter(s). A parameter can be measured by 1 or N sensor(s).
 
-This section contains information about the parameters measured by the glider or derived from glider measurements.
+This section contains information about the parameters measured by the glider or derived from glider measurements. The section is based on the fomrat used in
```

Comment:
typos. Corrected:
The section is based on the format used in Argo formats and enables parameter information interoperability with CMEMS.